src/components/common-components/dropdown-buttons/dropdown-buttons.component.tsx

```diff
@@ -33,16 +33,20 @@ export default function DropdownButtonsComponent({ buttons, mainLabel }: Options
         <ArrowDropDownIcon />
       </Button>
       <Popper open={open} anchorEl={anchorRef.current}>
+        <div className="display-main-button">
+          <Button variant="primary">
+            {mainLabel}
+            <ArrowDropDownIcon />
+          </Button>
+        </div>
```

Comment:
Patrząc na kod ten fragment może wydawać się trochę bez sensu, ale chodzi o to, żeby główny button ("Plik") nadal był wyświetlany po otwarciu poppera. Z-index to złe rozwiązanie, bo dezaktywuje buttony, które znajdują się pod spodem (żeby mieć dostęp do przycisków, które rozwinęliśmy popper musi być najwyższą warstwą). Natomiast blendowanie/ustawienia transparentności/opacity dotyczyłyby wszystkich elementów (prześwitywałoby wszystko ze spodu-np. kalendarz miesięczny, na co @Qwebeck wcześniej zwrócił uwagę). A docelowo tylko ten mainButton powinien być powyżej poppera.
Podsumowując: kod jest raczej brzydki, ale działa dobrze. Jeżeli są dostępne jakieś sposoby jak to zrobić lepiej, to jestem otwarta na propozycje ;)

Reply:
Jak dla mnie, to jak najbardziej może być:
1. Ten kod jest zawarty w osobnym komponencie, co oznacza, że dość łatwo jest go używać. 
2. Nie wygląda to źle, gdybyś ty o tym nie napisała, to nawet nie zwróciłbym na to uwagi, ponieważ nie widzę jak to może zaszkodzić :D
3. To działa i działa dobrze!